src/openrct2-ui/windows/TopToolbar.cpp

```diff
@@ -2469,7 +2471,8 @@ static money64 TryPlaceGhostSmallScenery(
     scenery_remove_ghost_tool_placement();
 
     // 6e252b
-    auto smallSceneryPlaceAction = SmallSceneryPlaceAction(loc, quadrant, entryIndex, primaryColour, secondaryColour);
+    auto smallSceneryPlaceAction = SmallSceneryPlaceAction(
+        loc, quadrant, entryIndex, primaryColour, secondaryColour, COLOUR_DARK_BROWN);
```

Comment:
For consistency, I think it would be better to also pass the tertiary colour here.